config/zones.json

```diff
@@ -385,10 +384,17 @@
     "_comment": "http://www.hydroquebec.com/generation/",
     "capacity": {
       "gas": 411,
-      "hydro": 36366,
-      "oil": 130
+      "hydro": 41859,
+      "oil": 130,
+      "wind": 3508,
+      "unknown": 1056,
+      "nuclear": 657
```

Comment:
IMO this change to Quebec should not be part of a change adding PJM info. There is of course the chain of imports, but following that logic we'd add information for every U.S. zone and the change would be unmanageably big.

Apart from that I would dispute adding nuclear which is permanently shut down.